### PR TITLE
FEAT: Support Mixtral-8x7B-v0.1 models

### LINKED ISSUE #751: Request for Support for Mistralai/Mixtral-8x7B-Instruct-v0.1 LLM
Dear maintainers of this awesome project,

link: https://mistral.ai/news/mixtral-of-experts/

I would like to submit a request for supporting the recently released Mixtral-8x7B-Instruct-v0.1 LLM by mistralai on Hugging Face. This LLM is particularly interesting due to its advanced reasoning and instruction following capabilities.

As a regular user of your project, I am excited about its convenience and ease of use, especially with its one-click deployment feature. However, right now, this LLM is not included in your supported models list. I believe this could benefit the wider community if implemented.

Please let us know if you need any further information or additional help from our side to make this happen. We appreciate any timely feedback from your end and look forward to working together towards making this project even more valuable.

Best Regards,
Kuan

### PR DESCRIPTION
**Pull request by Bojun-Feng**
Fix #751 

- [x] Update Model Family JSON
- [x] Update README
- [x] Update Docs

Unfortunately, my laptop is too small to run the model locally. That being said, this does look like a really interesting model when I looked it up. I would recommend people who are interested in the details of this model to check out [this blog post](https://mistral.ai/news/mixtral-of-experts/) by the Mistral team.

### CODE REVIEW

File: xinference/model/llm/llm_family.json [hunk 1]
```diff
@@ -2070,6 +2070,64 @@
       ]
     }
   },
+  {
+    "version": 1,
+    "context_length": 32768,
+    "model_name": "mistral-8x7b-instruct-v0.1",
```
Comment (ChengjieLi28) [inline xinference/model/llm/llm_family.json:2076]: The model name should be `mixtral`. And this model name is too long for SDK and cmdline users. I think `mixtral-instruct-v0.1` is ok.
Same as other places in the document.

File: xinference/model/llm/llm_family.json [hunk 2]
```diff
@@ -2070,6 +2070,64 @@
...
+          "Q5_K_S",
+          "Q5_K_M",
+          "Q6_K",
+          "Q8_0"
+        ],
+        "model_id": "TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF",
+        "model_file_name_template": "mixtral-8x7b-instruct-v0.1.{quantization}.gguf"
+      }
+    ],
+    "prompt_style": {
```
Comment (aresnow1) [inline xinference/model/llm/llm_family.json:2117]: Where did you find the template?

**Comment by ChengjieLi28** [comment]
@Bojun-Feng . I will take over this PR now. Thanks for your contribution for `Mixtral` chat model supports.

File: xinference/model/llm/llm_family.json (hunk 2 above)
Comment (Bojun-Feng) [reply xinference/model/llm/llm_family.json]: I went on the [Huggingface repo](https://huggingface.co/TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF/tree/main) and looked at the filenames.

**Review by aresnow1 — APPROVED** [review]

**Review by aresnow1 — APPROVED** [review]

**Pull request merged**